LibPack 3.5.3 creates invalid VS solution

It includes `Coin `and `coin `projects:


```
<Project Path="src/3rdParty/coin/src/Coin.vcxproj" Type="8bc9ceb8-8b4a-11d0-8d11-00a0c91bc942" Id="d901cfcd-7b9e-35e5-a6e5-1f31c345385f">

<Project Path="src/3rdParty/pivy/interfaces/coin.vcxproj" Type="8bc9ceb8-8b4a-11d0-8d11-00a0c91bc942" Id="3d9bdfbb-d444-3fda-925a-4bfd53649789">
```

VS refuses to open such solution, because it doesn't distinguish letter case:

`FreeCAD.slnx(1036,3): error MSB4025: Project name 'coin' already exists in the 'Root' solution folder.`

--- a/.gitmodules
+++ b/.gitmodules
@@ -13,5 +13,5 @@
 	branch = freecad-master
 [submodule "src/3rdParty/pivy"]
 	path = src/3rdParty/pivy
-	url = https://github.com/FreeCAD/pivy.git
-	branch = freecad-master
+	url = https://github.com/jacquesbeaurain/pivy.git
+	branch = jbfreecad-master

--- a/cMake/FreeCAD_Helpers/SetupCoin3D.cmake
+++ b/cMake/FreeCAD_Helpers/SetupCoin3D.cmake
@@ -150,7 +150,7 @@ macro(SetupBundledCoinPivy)
     set(PIVY_PACKAGE_OUTPUT_DIR "${PROJECT_BINARY_DIR}/Mod/pivy"
         CACHE PATH "Build-tree output directory for bundled Pivy" FORCE)
     add_subdirectory("${CMAKE_SOURCE_DIR}/src/3rdParty/pivy" "${CMAKE_BINARY_DIR}/src/3rdParty/pivy")
-    set_property(TARGET coin PROPERTY INSTALL_REMOVE_ENVIRONMENT_RPATH TRUE)
+    set_property(TARGET pivy_coin PROPERTY INSTALL_REMOVE_ENVIRONMENT_RPATH TRUE)
     if (NOT DEFINED PIVY_VERSION OR PIVY_VERSION STREQUAL "")
         message(FATAL_ERROR "Bundled Pivy did not define PIVY_VERSION")
     endif ()

--- a/cMake/FreeCAD_Helpers/SetupCoin3D.cmake
+++ b/cMake/FreeCAD_Helpers/SetupCoin3D.cmake
@@ -150,7 +150,7 @@ macro(SetupBundledCoinPivy)
     set(PIVY_PACKAGE_OUTPUT_DIR "${PROJECT_BINARY_DIR}/Mod/pivy"
         CACHE PATH "Build-tree output directory for bundled Pivy" FORCE)
     add_subdirectory("${CMAKE_SOURCE_DIR}/src/3rdParty/pivy" "${CMAKE_BINARY_DIR}/src/3rdParty/pivy")
-    set_property(TARGET coin PROPERTY INSTALL_REMOVE_ENVIRONMENT_RPATH TRUE)
+    set_property(TARGET coini PROPERTY INSTALL_REMOVE_ENVIRONMENT_RPATH TRUE)
     if (NOT DEFINED PIVY_VERSION OR PIVY_VERSION STREQUAL "")
         message(FATAL_ERROR "Bundled Pivy did not define PIVY_VERSION")
     endif ()

